coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -0,0 +1,237 @@
+package filodb.coordinator
+
+import akka.actor.{Actor, ActorRef, Props}
+import com.typesafe.config.Config
+import net.ceedubs.ficus.Ficus._
+import org.velvia.filo.RowReader
+import scala.concurrent.Future
```

Comment:
IMO imports should be

java.* (to see the hurt first)
space
scala.*
akka.*
third.party.*
filodb.*